### ISSUE TITLE
Re-enable binary logs

### ISSUE THREAD

**Issue by dougbu** [dotnet/aspnetcore#32909]
Should be able to re-enable binary logs in more CI jobs after VS 16.10 P3 is available on the build agents. Try that out on (say) Thursday, May 27.

/fyi @dotnet/aspnet-build

**Comment by Pilchie** [comment]
FYI @KirillOsenkov

**Comment by dougbu** [comment]
Example areas for new logs:
- [ ] internal and non-PR builds i.e. eliminate `$(Windows64LogArgs)` variables and set up the build steps in the `Windows_build` job with binary logs regardless of the context
- [ ] a Linux build job or two
- [ ] an OSX build job or two
- [ ] a test job per platform, use the `Helix_x64` job for Windows because we've had some problems there
- [ ] the `Source_Build` job

**Comment by dougbu** [comment]
I think we've got enough logs now in 'main' at least. Any reason to keep this issue open @dotnet/aspnet-build❔

**Comment by wtgodbe** [comment]
Nope, especially not now that we've got an opt-in switch on internal builds.
